python/runtime/optimize/optimize.py

```diff
@@ -384,6 +416,34 @@ def solve_model(model, solver):
     return np.array(result_values, dtype=np_dtype)
 
 
+def load_odps_table_to_data_frame(odps_table, load_schema_only=False):
+    from odps import ODPS
+    from odps.df import DataFrame
+    project, table = odps_table.split('.')
+    ak = os.environ.get("SQLFLOW_TEST_DB_MAXCOMPUTE_AK")
```

Comment:
env names like `SQLFLOW_TEST_DB_MAXCOMPUTE_AK` is used for unit test only. Try db connection credentials from `db_conn_str` or `datasource`?

Reply:
Thanks for your advice! Fixed.